### ISSUE TITLE
Drop support for depositing individual submissions

### ISSUE THREAD

**Issue by jonasraoni** [pkp/pln#77]
Besides depositing full issues, the plugin also supports depositing individual submissions, which depends on the value of the `object_type` setting.

**Problem:**
- The setting isn't configurable, and its default value is to deposit issues. Therefore, it's basically alive just to support a legacy workflow, which will end up not being well tested and lead to zombie code.
- The current implementation attempts to group submissions in groups of 20 for each deposit. If the threshold isn't met, the deposit will never be created.

**Notes:**
- At the Preservation Network there are currently 571 deposits out of 137790 (0.4%), that **perhaps** are using this setting (they have neither an issue nor a volume number).
- Dropping this code will also allow merging the database entities of the plugin into a single one.
- This task should also cover the #63

**Comment by kaitlinnewson** [comment]
Since we will support publishing without issues/continuous publishing models, we will need to keep the deposit of individual articles as they may not be in an issue.

**Comment by jonasraoni** [comment]
Yep, this issue is outdated, we were already discussing how to implement the continuous publishing (probably JATS + alternative file for the issues).
Anyway, the code which supports this should be wiped out, just like the other confuse things (e.g. set of status using binary flags, status management, ...).
